ldk/node/src/hostClients/browserService.ts

```diff
@@ -1,11 +1,15 @@
 import { StoppableStream, StreamListener } from './stoppables';
 
-export interface BrowserSelectedTextResponse {
+export interface IBrowserSelectedTextResponse {
   text: string;
   url: string;
   tabTitle: string;
 }
 
+export type BrowserSelectedTextResponse =
```

Comment:
I see we're doing this in a few places. If we think that sometimes `undefined` values are being returned I'd like to understand when that can happen, and whether it makes sense fo surface these Response Or Undefined types at this level as opposed to within the `BaseClient` or some other level of abstraction.

Reply:
Yeah, how I did here is just plain wrong, in hindsight. Honestly, what we likely should be doing is rejecting if something comes back as undefined in most of these cases. As in, gRPC will always set things to "optional", but at a layer above that, we have cases where we really do consider a field required enough for it to be an error. In any case, we'll deal with this when we're better prepared - not bumping the minor version for the moment.